dcos/errors.py

```diff
@@ -49,11 +49,17 @@ class DCOSAuthenticationException(DCOSHTTPException):
 
     :param response: requests Response object
     :type response: Response
+    :param message: An optional message for the exception.
```

Comment:
Should we make it more obvious that this message will actually be exposed to the user?